report/report-assets.js

```diff
@@ -7,14 +7,20 @@
 
 const fs = require('fs');
 
+/* eslint-disable max-len */
 const REPORT_TEMPLATE = fs.readFileSync(__dirname + '/assets/standalone-template.html', 'utf8');
 const REPORT_JAVASCRIPT = fs.readFileSync(__dirname + '/../dist/report/standalone.js', 'utf8');
 const REPORT_CSS = fs.readFileSync(__dirname + '/assets/styles.css', 'utf8');
+const REPORT_FLOW_TEMPLATE = fs.readFileSync(__dirname + '/assets/standalone-flow-template.html', 'utf8');
```

Comment:
including in `report-assets` and `report-generator` _may_ mean the flow report gets included in all bundles of the report generator since we don't really tree shake anything

Reply:
For the DT bundle, the assets are not included unless we write them to a specific directory under `dist`: https://github.com/GoogleChrome/lighthouse/blob/98eebdaf6daa82957cadd057b16ce680af226bc3/build/build-dt-report-resources.js#L33-L37

Reply:
LR bundle does include flow report assets

Reply:
I moved the flow assets to a separate file so we can easily ignore them for the LR bundle.